src/drivers/general/queries/IrqlNotSaved/IrqlNotSaved.ql

```diff
@@ -59,7 +61,7 @@ class IrqlFlowConfiguration extends DataFlow::Configuration {
 }
 
 /**
- * Represents a "fundamental" function that restores IRQL, i.e. one defined
+ * A "fundamental" function that restores IRQL, i.e. one defined
```

Comment:
What do the quotes around the word fundamental mean?

Reply:
I used the quotes here to indicate that I was using 'fundamental' in a slightly unique way that I defined in the next clause, and that this was not a standardized definition of 'fundamental'.  But it's probably a little confusing.  I'll see if I can rewrite.